src/main/java/org/opensrp/web/rest/SearchResource.java

```diff
@@ -83,9 +81,13 @@ public List<Client> search(HttpServletRequest request) throws ParseException {//
 			String attributeType = StringUtils.isBlank(attributes) ? null : attributes.split(":", -1)[0];
 			String attributeValue = StringUtils.isBlank(attributes) ? null : attributes.split(":", -1)[1];
 			
-			attributeMap = new HashMap<String, String>();
+			attributeMap = new HashMap<>();
 			attributeMap.put(attributeType, attributeValue);
 		}
+		if (phoneNumber.isPresent()) {
+			attributeMap = new HashMap<>();
+			attributeMap.put("alt_phone_number", phoneNumber.get());

```

Comment:
Move the literal string to constants

Reply:
This means I will have to deploy a new version of server common